tests/tests_geomstats/test_visualization_beta_distributions.py

```diff
@@ -0,0 +1,90 @@
+"""Unit tests for visualization."""
```

Comment:
Rename file name to test_visualization_beta.py
Change docstring to Unit tests for visualization of beta distributions.